### ISSUE TITLE
Search is broken

### ISSUE THREAD

**Issue by crandmck** [c2pa-org/specifications#65]
The search feature is totally non-functional.  Not sure when this started, but loading any page on the rendered site shows some 404 errors trying to load `search-index.js` and `search.css` which presumably is the cause (or at least related) to the problem:

<img width="1074" alt="Screenshot 2024-09-24 at 2 02 28 PM" src="https://github.com/user-attachments/assets/859c34d6-4260-49ae-a419-01dce2c004c8">

**Comment by lrosenthol** [comment]
Yes, there is an internal issue tracking this as well - but thanks for reporting!!
